product_margin_classification/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,35 @@
+This module is designed to extend Sale Price computation in Odoo.
+
+This module add a new model 'Margin Classifications' linked to Product Templates.
+
+A margin classification has a 'Profit Margin' field and extra fields to manage
+computation method, like in Pricelist Item model (Markup Rate, Rounding and Surcharge fields)
+
+If product has a margin classification defined and the theoretical price is
+not the same as the sale price, an extra field 'Theoretical Price' is
+displayed, based on the Margin Classification and a button is available to
+change sale price.
+
+.. image:: /product_margin_classification/static/description/product_template_form.png
```

Comment:
could your replace by : 

`` .. image:: ../static/description/product_template_form.png``

(3 occurencies)

It will work in all cases. (apps store, github, etc...)

Reply:
@MPoch-PlanetaTIC please, replace the other 2 occurencies, thanks